src/main/java/com/github/romankh3/image/comparison/ImageComparisonUtil.java

```diff
@@ -45,19 +50,24 @@ public static BufferedImage readImageFromResources(String path) throws ImageComp
         File imageFile = new File(path);
         if (imageFile.exists()) {
             try {
+                logger.info("Read image form User Successfully.");
```

Comment:
Just because it's a library, we need to move lower logger level for it.
For example: info -> debug